lib/erb_lint/linters/rubocop.rb

```diff
@@ -131,11 +131,17 @@ def tempfile_from(filename, content)
       end
 
       def rubocop_processed_source(content, filename)
-        ::RuboCop::ProcessedSource.new(
+        source = ::RuboCop::ProcessedSource.new(
           content,
           @rubocop_config.target_ruby_version,
           filename
         )
+        if ::RuboCop::Version::STRING.to_f >= 1.38
+          registry = RuboCop::Cop::Registry.global
+          source.registry = registry
+          source.config = @rubocop_config
+        end
```

Comment:
Can we do the check on feature rather than version?
```suggestion
        source.registry = RuboCop::Cop::Registry.global if source.respond_to?(:registry=)
        source.config = @rubocop_config if source.respond_to?(:config=)
```

Reply:
Well, the two things go together so it doesn't seem sensible to have two conditions.

Also, there is already a version check in that file: https://github.com/Shopify/erb-lint/blob/9b7dec10f51cd5e505347aded25d4d7e1aec94c4/lib/erb_lint/linters/rubocop.rb#L39, so it seems consistent to handle this the same way.

Finally, if ever in the distant future erb-lint supports only RuboCop 1.38 and above, it will be easier to see that the condition can be removed.

Reply:
Fair enough